pkgs/servers/sql/mariadb/default.nix

```diff
@@ -121,7 +122,8 @@ everything = stdenv.mkDerivation (common // {
   buildInputs = common.buildInputs ++ [
     xz lzo lz4 bzip2 snappy
     libxml2 boost judy libevent cracklib
-  ] ++ optionals (stdenv.isLinux && !stdenv.isArm) [ numactl ];
+  ] ++ optional (stdenv.isLinux && !stdenv.isArm) numactl
+    ++ optional stdenv.isDarwin libiconv;
```

Comment:
`libiconv` probably belongs to `common`, not just `everything`.

Reply:
That also worked, just did the minimal stuff to get it working.

Reply:
It was not sufficient because `mariadb.client` was left broken. I have fixed it in 306eefaa95aa4be433e126aa1c3f9e04ea5b8430 .

Reply:
Thanks, I didn't to check the client.